config/initializers/audit_events.rb

```diff
@@ -8,4 +8,5 @@
 
 Rails.application.reloader.to_prepare do
   Audit::Subscribe.listen(:moderator, :internal) unless Rails.env.test?
+  Audit::Subscribe.listen(:admin_api, :internal) unless Rails.env.test?
```

Comment:
It took me longer than I'm proud to admit to find this line in our codebase 😅 

But I'm happy I learned more about how the [ActiveSupport Instrumentation API](https://guides.rubyonrails.org/active_support_instrumentation.html#creating-custom-events) works!